<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datostipofijo</class>
 <widget class="QDialog" name="datostipofijo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos tipo fijo</string>
  </property>
  <widget class="QFrame" name="frame_datosfijo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QTableWidget" name="tableWidget_datosfijo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1001</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Revisión</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Año</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nº Cuota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cuota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Interes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amortización</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Capital pendiente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fijo(%)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pago</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_totalesfijo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QTableWidget" name="tableWidget_fijototales">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>TOTALES :</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Total interes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total amortizado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total pagado</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_expexceldatfijo">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>390</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar a excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_salirformdatosfijo">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>390</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_graf_fijo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_grafFijo"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COSTE DE LA HIPOTECA</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
